make-util.js

```diff
@@ -174,12 +174,12 @@ var buildNodeTask = function (taskPath, outDir) {
             fail('The package.json should not contain dev dependencies other than typescript. Move the dev dependencies into a package.json file under the Tests sub-folder. Offending package.json: ' + packageJsonPath);
         }
 
-        run('npm ci');
+        run('npm install');
     }
 
     if (test('-f', rp(path.join('Tests', 'package.json')))) {
         cd(rp('Tests'));
-        run('npm ci');
+        run('npm install');
```

Comment:
Can you please explain why you are changing this?

Reply:
`npm ci `doesn't work when building some tasks, it was originally `npm install`, and changed to `npm ci` when migrating app center related tasks in this [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/19212), so I think we should revert back to `npm install`.